README.md

```diff
@@ -934,6 +934,20 @@ clusters to be UC compatible.
 
 [[back to top](#databricks-labs-ucx)]
 
+## `lint-local-code` command
+
+```text
+databricks labs ucx lint-local-code
+```
+
+Once [table migration](#table-migration-workflow) is complete, you can run this command to 
+assess all migrations required in a local directory or a file. It only takes seconds to run and it
+gives you an initial overview of what needs to be migrated without actually performing any migration.
+
+This command detects all dependencies, and analyzes them. It is still experimental and at the moment only supports Python and SQL files.
```

Comment:
can you attach the example of command to PR description and/or documentation? it's not clear how results look like and how to interpret them.

Reply:
added to the PR. I believe it's self-explanatory ?